modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/commands/DownloadCommand.java

```diff
@@ -19,7 +19,7 @@
 
 import com.liferay.portal.tools.bundle.support.constants.BundleSupportConstants;
 import com.liferay.portal.tools.bundle.support.internal.util.FileUtil;
-import com.liferay.portal.tools.bundle.support.internal.util.HttpUtil;
+import com.liferay.portal.tools.bundle.support.util.HttpUtil;
```

Comment:
Can you switch this to use DownloadCommand instead of making HttpUtil public?  DownloadCommand should bee able to be used to download a single file.